extensions/amp-story/1.0/amp-story-open-page-attachment.js

```diff
@@ -72,11 +89,19 @@ export const buildOpenInlineAttachmentElement = (element) =>
 export const renderPageAttachmentUI = (win, pageEl, attachmentEl) => {
   const openImgAttr = attachmentEl.getAttribute('cta-image');
   const attachmentHref = attachmentEl.getAttribute('href');
-  if (isPageAttachmentUiV2ExperimentOn(win) && !attachmentHref && openImgAttr) {
-    return renderPageAttachmentUiWithImages(win, pageEl, attachmentEl);
-  } else {
-    return renderDefaultPageAttachmentUI(pageEl, attachmentEl);
+  if (isPageAttachmentUiV2ExperimentOn(win)) {
```

Comment:
Can you change this with an early return that builds the default page attachment UI, to avoid nested if statements?

Reply:
How's this?

`
if (isPageAttachmentUiV2ExperimentOn(win) && attachmentHref) {
    return renderOutlinkPageAttachmentUI(
      win,
      pageEl,
      attachmentEl,
      attachmentHref
    );
  } else if (isPageAttachmentUiV2ExperimentOn(win) && openImgAttr) {
    return renderPageAttachmentUiWithImages(win, pageEl, attachmentEl);
  } else {
    return renderDefaultPageAttachmentUI(pageEl, attachmentEl);
  }
`

But I don't think an early return for the default case will work bc default is used either when 1) the flag is not on, or 2) when the flag is on, and it's an inline attachment with no provided images. The UX for that state is the same as the old one.